app/controllers/journalist/articles_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Journalist::ArticlesController < ApplicationController
+    before_action :check_journalist, only: [:create, :new]
+    before_action :authenticate_user!, only: [:create, :new]
+
+    def new
```

Comment:
It makes no sense to display all articles on the `new` action.

Reply:
And shouldn't a journalist be able to only see his own articles?

Reply:
i'll move it to index. journalist seeing only his own is a later user story.

Reply:
Per here is the user story
```
As a Journalist,
In order to change articles I have written
I would like to be able to edit or delete articles
```
If you look at line 2 it explicitly says he should be able to change his articles.

And you should not be fetching all articles in the new method if you want to stay restful.

Reply:
done